portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,155 @@
+
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>hey, it's malachi 🚀</title>
+        <!-- <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css"> -->
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+        <link rel="stylesheet" href="style.css">
+        <script src="https://use.fontawesome.com/3e86b2749f.js"></script>
+        <link rel="shortcut icon" href="images/moon.png"/>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+        <!-- <script src="https://unpkg.com/aos@next/dist/aos.js"></script> -->
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <!-- HEADER -->
+        <header>
+            <nav class="top-nav">
+                <ul class="nav-list">
+                    <li>
+                        <a href="index.html" class="nav-link">home</a>
+                    </li>
+                    <li>
+                        <a href="#biography" class="nav-link">biography</a>
+                    </li>
+                    <li>
+                        <a href="#projects" class="nav-link">projects</a>
+                    </li>
+                     <li>
+                        <a href="https://drive.google.com/file/d/1KUyDlDXmT-jwwOFXAHw0-48sdiVLewFJ/view" class="nav-link">resume</a>
+                    </li>
+                     <li>
+                        <a href="http://linkedin.com/in/malachiblenman" class="nav-link">linkedin</a>
+                    </li>
+                     <li>
+                        <a href="http://github.com/malachiblenman" class="nav-link">github</a>
+                    </li>
+                     <li>
+                        <a href="https://www.instagram.com/mallyblenman/" class="nav-link">instagram</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class= "about-me-pic"> 
+                <img id="selfie" src="images/mally2.JPG">
+                 <img id="moon" src="images/moon.png">
+                </div>
+            <div class="landing-text">
+              <h1>malachi blenman
```

Comment:
In general I haven't often seen `<p>` tags inside of an `<h1>` so I'm not sure if it is good practice. An alternative if you want the following lines to be large as well is to put them each in an `<h1>` or `<h2>` which should have similar spacing.

Reply:
Done! I utilized <br> instead since I wanted them to be the same size, and it worked perfectly!